README.md

```diff
@@ -1 +1,17 @@
-# phase-0-gps-1
\ No newline at end of file
+git clone https://github.com/aazevedojr/phase-0-gps-1.git
+Saves a copy of the repo from github to your computer
+
+git status
+Tells the status of all files within the repo
+
+git add awesome_page.md
+Adds files to the staging area so that they can be commited
+
+git commit -m "Add file for Release 1"
+Creates a save point for all staged changes
```

Comment:
Nice description! Commits, or "save points", are great to roll back to in case we accidentally merge something that's broken :)